src/helpers.js

```diff
@@ -240,20 +234,23 @@ export async function processESLintMessages(response, textEditor, showRule, work
         range = generateRange(textEditor, msgLine, msgCol)
       }
       ret = {
-        filePath,
-        type: severity === 1 ? 'Warning' : 'Error',
-        range
+        severity: severity === 1 ? 'warning' : 'error',
+        location: {
+          file: filePath,
+          position: range
+        }
       }
 
       if (showRule) {
-        const elName = ruleId ? 'a' : 'span'
-        const href = ruleId ? ` href="${ruleURI(ruleId).url}"` : ''
-        ret.html = `${escapeHTML(message)} (<${elName}${href}>${ruleId || 'Fatal'}</${elName}>)`
+        const url = ruleId ? ruleURI(ruleId).url : ''
+        // TODO: fetch rule Markdown from eslint
+        ret.url = url
```

Comment:
Since Linter now has built in handling of URL references, this should _always_ be specified if it is available. The `showRule` option should now only control whether the ` (rule-name)` text is added to the `excerpt`.